onadata/libs/templates/registration/verification_email.txt

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% blocktrans with expiration_days=expiration_days username=username expiration_days_p=expiration_days|pluralize %}
+Hi {{ username }},
+
+To verify your email please click on the link below:
```

Comment:
Can you start a language discussion on this with the product team internally, I think it may need some improvement unless this is exactly the same as the default done by the registration app.

Reply:
I agree with you that this might need improvement. I'll start the discussion. The template was mainly used to check that the email functionality works.